File: daigo/fizzbuzz.php
<?php
// fizz buzz
// ex) $ php fizzbuzz.php 10
//     $ buzz

$num = $argv[1];

if ( ($num % 3) == 0 ) {
    echo "fizz ";
}
if ( ($num % 5) == 0 ) {
    echo "buzz ";
}
echo "\r\n";

File: torufuru/fizzbuzz.py
def fizzbuzz(i):
    for i in range(1,i):
        fb = ""
        if i % 15 == 0:
            fb = "fizzbuzz"
        elif i % 3 == 0:
            fb = "fizz"
        elif i % 5 == 0:
            fb = "buzz"

        print "%s: %s" % (i, fb)

if __name__ == '__main__':
    import sys
    fizzbuzz(int(sys.argv[1]))
